src/xcode/ENA/ENA/Source/Models/Task Scheduling/ENATaskScheduler.swift

```diff
@@ -22,14 +22,17 @@ import UIKit
 enum ENATaskIdentifier: String, CaseIterable {
 	// only one task identifier is allowed have the .exposure-notification suffix
 	case primaryBackgroundTask = "exposure-notification"
```

Comment:
why the odd naming of primary and secondary? Why not give them an actual descriptive name? like the `fetchTestResultsTask`.